data/zilliqa/assets/zil1lq3ghn3yaqk0w7fqtszv53hejunpyfyh3rx9gc/meta.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Zilliqa",
+  "symbol": "ZIL",
+  "web": "https://elons.io/",
+  "email": "blockchainandy@hotmail.com",
+  "publicTeam": true,
+  "product": false,
+  "explorer": "https://viewblock.io/zilliqa/address/zil1lq3ghn3yaqk0w7fqtszv53hejunpyfyh3rx9gc",
+  "links": {
+    "github": "Token-Elon",
```

Comment:
Nothing on the github besides a fork, remove for now

Reply:
ZRC contract repos are coming shortly, will remove for no, thanks